src/workloads/execution/LastPoint.yml

```diff
@@ -0,0 +1,195 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of last point queries after the first milestone.
+
+Actors:
+- Name: CreateTimeSeriesCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Operation:
+      OperationMetricsName: CreateTimeSeriesCollection
+      OperationName: RunCommand
+      OperationCommand:
+        {create: &coll Collection11, timeseries: {timeField: "time", metaField: "tags"}}
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Nop: true
+  - Repeat: 1
+    Database: *db
+    Collection: *coll
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 5000000
+    BatchSize: &batchSize 100
+    Document:
+      time: {^RandomDate: {min: "2022-01-01", max: "2022-01-01T01:00:00.001"}}
```

Comment:
For my understanding, could you clarify the range of this - does this mean that all 5 million docs will have a time value within the same hour?

Reply:
Yes, I did this to model a use case with an incredibly high amount of docs in a short range of time. But I can modify this to better suit usage models if needed!